### ISSUE TITLE
Header structure for Reusable Q&As

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#15717]
## Describe the defect
When reusable Q&As are added to content, the editor has the ability to add an overall Header to the group. This header is not required which causes the overall heading structure on the page to be incorrect in certain situations.

### Scenarios
**Reusable Q&As set as accordions with Header**
- H2: Header
  - H2: Each question is an accordion title WRONG  - should be H3

**Reusable Q&As set as accordions without Header**
- H2: Each question is an accordion title

**Reusable Q&As no accordions with Header**
- H2: Header
  - H3 Each question is an accordion title

**Reusable Q&As no accordions without Header**
- H3: Each question is an accordion title WRONG  - should be H3

## To Do
What we need to do it dynamically set these headers to the proper level based on if there is a header or not. We also need to confirm if H2 would always be the correct start heading. For example, would these ever be underneath a different H2, in which case we should be working with H3s and H4s?

## AC / Expected behavior

- [ ] Headings are correct within the Reusable Q&As where they appear: 
  - [ ] FAQs
  - [ ] Resources & Support
  - [ ] Benefit Detail Pages
  - [ ] CLPs
- [ ] Overall heading structure is correct on the page
- [ ] Accessibility review

## Additional context
- Josh did a lot of work setting the proper headings for accordions last year, do we need to do something similar for Q&As?
- See this ticket for examples of prior work done: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/10126

### Team
Please check the team(s) that will do this work.

- [ ] `CMS Team`
- [x] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

**Comment by jilladams** [comment]
Added list of products where reusable Q&A appear to ACs.

**Comment by chriskim2311** [comment]
@laflannery Hi! This is some notes I had while working on this ticket. Would like to go over them with you when you are back. 

**Reusable Q&As set as accordions with Header**
H2: Header
H2: Each question is an accordion title WRONG - should be H3
- This seems to be fixed showing as H3 with the header. 

**Reusable Q&As no accordions without Header**
H3: Each question is an accordion title WRONG - should be H3
-Question: Should these be H2 not H3?

Tugboat for testing: https://main-dyel3cg3kniaoc6nrizjhqazkfgueref.demo.cms.va.gov/node/64397/
https://web-dyel3cg3kniaoc6nrizjhqazkfgueref.demo.cms.va.gov/resources/test-faq-page/

**Comment by laflannery** [comment]
@chriskim2311 I just went over all 4 scenarios again using the page you gave me above and you're right, they don't match up with what I have in the original ticket description. Only scenario 2 is now wrong (see below). I'm going to look at a few more existing pages to see if this is the case on existing pages.

**Reusable Q&As set as accordions with Header**

H2: Header
H3: Each question is an accordion title CORRECT

**Reusable Q&As set as accordions without Header**

H3: Each question is an accordion title - WRONG: should be H2s

**Reusable Q&As no accordions with Header**

H2: Header
H3 Each question is an accordion title - CORRECT

**Reusable Q&As no accordions without Header**

H2: Each question is an accordion title - CORRECT

**Comment by laflannery** [comment]
Update:

I think the different content types are using different templates. Example:
- The test page you created above is an FAQ page and has the scenarios I noted
- I found an existing [resources and support detail page](https://www.va.gov/resources/creating-an-account-for-vagov/) and it has scenario 2 (Reusable Q&As set as accordions without Header) and it follows the patterns in the original ticket description NOT the pattern on the FAQ page - which is that each accordion is an H2. 

So I think what we might need to do is create a test page for each of the 4 content types indicated in the AC (FAQ, Resources and support, Benefits Detail, CLPs) and confirm what each are doing and how they all might need ot be fixed. Does that seem right?

**Comment by chriskim2311** [comment]
@laflannery Thanks for the feedback! I will work on this in the next sprint.

**Comment by chriskim2311** [comment]
@laflannery Can you take a look at prod for this change? Did you know a page where this was happening I can also take a look thanks!

**Comment by laflannery** [comment]
@chriskim2311 This looks good, Here are a couple links I found:

- https://www.va.gov/resources/the-pact-act-and-your-va-benefits/
- https://www.va.gov/resources/getting-urgent-care-at-va-or-in-network-community-providers/ (the bottom section is reusable Q&A)

The other 2 types (Benefits hubs and CLPs) were only just recently updated to include these so I don't think any content on prod has these yet.

**Comment by jilladams** [comment]
Closing based on verification of the content we can find. We can open additional tickets for anything we find on the content types without prod reusable Q&A content, later on.
